packages/mobile/nodejs-assets/override-dlopen-paths-data.json

```diff
@@ -0,0 +1 @@
+[{"originalpath":["node_modules","classic-level","build","Release","classic_level.node"],"newpath":["..","Frameworks","classic-level.framework","classic-level"]}]
```

Comment:
weird you don't have and entry for one for leveldown.node

Reply:
huh of course you're right, but I saw it working on my iPhone

Reply:
:shrug:

Reply:
Ok, it looks like the app doesn't care about the path for leveldown (strange). I'll commit the file containing proper mapping for it anyway